api/async/publish_test.ts

```diff
@@ -426,3 +424,56 @@ Deno.test({
     }
   },
 });
+
+Deno.test({
+  name: "publish too large",
+  async fn() {
+    try {
+      const id = await database.createBuild({
+        options: {
+          moduleName: "ltest_big",
+          ref: "0.0.1",
+          repository: "luca-rand/testing_big",
+          type: "github",
+          version: "0.0.1",
+        },
+        status: "queued",
+      });
+
+      await handler(
+        createSQSEvent({ buildID: id }),
+        createContext(),
+      );
+
+      assertEquals({ ...await database.getBuild(id), created_at: undefined }, {
+        created_at: undefined,
+        id,
+        options: {
+          moduleName: "ltest_big",
+          ref: "0.0.1",
+          repository: "luca-rand/testing_big",
```

Comment:
It would be good to have these test repos under denoland org at some point... but let's worry about that later.